src/data.service.ts

```diff
@@ -1,47 +1,48 @@
-import { users } from './users';
+import { UsersModel } from './user.model';
+import { Op } from './data-access';
 
-export const addUser = (login: string, password: string, age: number) => {
-    const id = getNextId();
-    users.push({ login, password, age, id, isDeleted: false });
-    return id;
-};
+export default class DataService {
```

Comment:
As it is responsible for the particular entity, namely the user entity it must be located within a specific file like: `user.service`